DataFormats/Detectors/PHOS/include/DataFormatsPHOS/Digit.h

```diff
@@ -50,7 +47,7 @@ class Digit : public DigitBase
   /// \brief Replace content of this digit with new one, from hit
   /// \param PHOS Hit
   /// \return
-  void FillFromHit(Hit hit);
+  void fillFromHit(Hit hit);
 
   /// \brief Comparison oparator, based on time and absId
   /// \param another PHOS Digit
```

Comment:
since you have at most 12k cells, you can save 4 bytes in the size of this class by defining it as 
````
  int mLabel;       ///< Index of the corresponding entry/entries in the MC label array
  float mAmplitude; ///< Amplitude
  float mTime;      ///< Time
  short mAbsId;       ///< cell index (absolute cell ID)
  bool mIsHighGain; ///< High Gain or Low Gain channel (for calibration)
````